static/js/newsletter_signup.js

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
```

Comment:
When adding `'use strict';` here it will apply globally, so you need to be a bit careful that this script is not concatenated with any other scripts (which are not strict). This might be fine here currently, but it's common practice to use the function form of `'use strict';` as a safety measure (so wrap everything here in a function that can safely be considered *strict*).